src/lib/string_utils.adb

```diff
@@ -87,13 +86,10 @@ is
    -- Incr --
    ----------
 
-   function Incr (Ptr : Pointer) return Pointer
+   function Incr (Ptr : Valid_Pointer) return Valid_Pointer
    is
-      Next : Pointer := Ptr + 1;
+      Next : constant Valid_Pointer := Ptr + 1;
```

Comment:
Any reason you're not just returning `Ptr + 1` (maybe in an expression function)?

Reply:
This is probably because of the check I did before that was removed with this commit. You're right, I could directly return `Ptr + 1`.